DQMOffline/Configuration/test/BuildFile.xml

```diff
@@ -4,12 +4,12 @@
 <test name="GetTestDQMOfflineConfigurationFile" command="edmCopyUtil ${INFILE} $(LOCALTOP)/tmp/"/>
 
 <!-- To make the tests run in parallel, we chunk up the work into arbitrary sets of 10 sequences. -->
-<test name="TestDQMOfflineConfiguration" command="runtests.sh   ${step_value} ${value} file://${LOCALTOP}/tmp/${INFILE_NAME}" for="0,320,10">
+<test name="TestDQMOfflineConfiguration" command="runtests.sh   ${step_value} ${value} file://${LOCALTOP}/tmp/${INFILE_NAME}" for="0,329,10">
```

Comment:
@smuzaffar What is `329`? It looks like a magic number in my eyes.

Reply:
not a magic number. Currently we have `321` sequences returned by `cmsswSequenceInfo.py --runTheMatrix --steps DQM,VALIDATION` which should be processed by `TestDQMOfflineConfiguration_NN` unit tests  and `TestDQMOfflineConfigurationGotAll` should not process any sequences (otherwise it should fail). 

I could have set this number to `321` but I decided to set it to `329` so that if new extra sequences are added then they can be processed by  `TestDQMOfflineConfiguration_NN` untill we hit `330` sequences